gui/mainwindow.cpp

```diff
@@ -836,35 +836,50 @@ void MainWindow::addIncludeDirs(const QStringList &includeDirs, Settings &result
     }
 }
 
-Library::Error MainWindow::loadLibrary(Library &library, const QString &filename)
+Library::Error MainWindow::loadLibrary(Library &library, const QString &filename, bool verbose)
 {
     Library::Error ret;
 
     // Try to load the library from the project folder..
     if (mProjectFile) {
         QString path = QFileInfo(mProjectFile->getFilename()).canonicalPath();
-        ret = library.load(nullptr, (path+"/"+filename).toLatin1());
+        QString libpath = path+"/"+filename;
+        if (verbose)
+            std::cout << "looking for library '" + libpath.toStdString() + "'" << std::endl;
```

Comment:
I'm not sure that the GUI should write to std::cout it's not very likely that a user would see this info.

Reply:
It is disabled by default and cannot be enabled by the user. It just is for me so I can compare the output. The whole code will go away when I am done with it.